addons/popochiu/popochiu_plugin.gd

```diff
@@ -102,7 +103,7 @@ func _enter_tree() -> void:
 	
 	# Fill the dock with Rooms, Characters, Inventory items, Dialogs and
 	# AudioCues
-	main_dock.grab_focus()
+	main_dock.call_deferred('grab_focus')
```

Comment:
Why do you need to defer this call here?

Reply:
I probably did this as an attempt to solve #17 . But what really made the trick at the end was making a deferred call to `maind_dock.fill_data()`.

Reply:
... that I spotted below. OK then, I guess you could leave this deferred. This is another topic I need to grasp better, I admit.
Thanks for your answer anyway.